build.gradle

```diff
@@ -82,3 +85,94 @@ githubRelease.doFirst {
     assets = [ filename ]
   }
 }
+
+
+// setup yaml rest tests
+testClusters {
+  yamlRestTest
+}
+
+sourceSets {
+  yamlRestTest
+}
+
+configurations {
+  yamlRestTestImplementation.extendsFrom testImplementation
+  yamlRestTestRuntimeOnly.extendsFrom testRuntimeOnly
+  restTestSpecs
+}
+
+tasks.register('copyRestTestSpecs', Copy) {
+  from zipTree(configurations.restTestSpecs.singleFile)
+  into "$buildDir/restResources/restspec"
+}
+
+TaskProvider<Zip> bundle = project.getTasks().withType(Zip.class).named("bundlePlugin");
+
+// Register rest resources with source set
+sourceSets.yamlRestTest.getOutput().dir("$buildDir/restResources/restspec");
+
+tasks.register('yamlRestTest', StandaloneRestIntegTestTask) { testTask ->
+    testTask.dependsOn(bundle, 'copyRestTestSpecs')
+
+    def cluster = testClusters.yamlRestTest
+    cluster.plugin(bundle.flatMap(AbstractArchiveTask::getArchiveFile))
+    testTask.useCluster(testClusters.yamlRestTest)
+
+    testTask.mustRunAfter(project.getTasks().named("test"))
+    testTask.setTestClassesDirs(sourceSets.yamlRestTest.getOutput().getClassesDirs())
+    testTask.setClasspath(sourceSets.yamlRestTest.getRuntimeClasspath())
+
+
+    SystemPropertyCommandLineArgumentProvider nonInputProperties = new SystemPropertyCommandLineArgumentProvider()
```

Comment:
@breskeby this will be unnecessary here as well right since in 7.14.1 we'll auto apply this to all test tasks, yes?